news/2021-03-02-mappers-report-february.md

```diff
@@ -0,0 +1,114 @@
+
+Even if you prefer metal as a type of element rather than music, fret not. A glimpse at the six-feet deep section of Val's mapping career clearly shows that anything is possible in the music department too. Regardless, this is a name you don't want flying under your radar in the future if fluid streams or beautiful sliderwork are close to your heart.
+
+### Smokebloke
+
+Soon on the rise, [Smokebloke](https://osu.ppy.sh/users/7535045) *(Calliecube, Obamacube, or whatever else you may know them by)* has been making moves in the mapping scene and has improved their mapping to a very substantial level. Soon to rank their first beatmap, [Toromaru - Shounen wa Sora o Tadoru Prog Piano Remix](https://osu.ppy.sh/beatmapsets/1197597) as well as having their [NIWASHI - Playing With Ruby](https://osu.ppy.sh/beatmapsets/1178229) beatmap put up for the Loved voting at the very moment, Smokebloke certainly knows their way around the editor.
+
+While these projects are on the older side, their newer beatmaps only continue to improve in quality, such as [Trerey-U+NIWASHI - Hisui](https://osu.ppy.sh/beatmapsets/1303715) with its dense rhythm and solid structure, providing both a feast for your eyes but also a fun experience while playing.
+
+If you prefer technical finger control focused maps with pretty visuals, you should check out all of these and more to come from Smokebloke!
+
+## <a id="mapping-contests"></a>Mapping Contests
+
+Do you like mapping? Do you seek competition? Do you want osu!supporter or even a shiny new badge? Search no more! All this and much more is what you can get in this month's contests! So what are you waiting for? Go and sign up!
+
+### Monthly Speedmapping Series 2021
+
+A shiny new monthly contest rises on the horizon! It's the [**Monthly Speedmapping Series 2021**](https://osu.ppy.sh/community/forums/topics/1243245) or in short **MSS** for the osu! game mode.
+
+If you like creating beatmaps and are also a fan of speedruns in general, this is for you! In this monthly contest, you will have to create a single difficulty beatmap for the osu! game mode from two provided songs in no more than 3 hours! This contest will go through the entire year, and every time you participate you will be able to gather points. At the end of the year, the top 3 will receive prizes such as osu!supporter and a shiny new badge for your profile page!
+
+To participate, you will need to join the [Discord server](https://discord.gg/fVgU9pA "Discord"). For the contest rules and further information, feel free to check out the forum thread linked above.
+
+May the best and the fastest mapper win!
+
+<a id="conclusion"></a>
+
+---
+
+And that's it for this month's issue of the Mappers' Report! A short issue this time as not much happened.
```

Comment:
```suggestion
And that's it for this month's issue of the Mappers' Report! This one was a bit shorter than usual because February was a slow month.
```